spec/factories/classifications.rb

```diff
@@ -11,11 +11,11 @@
                  {task: "another_one",
                   value: [1, 2]}]
     user_ip "192.168.0.1"
-    workflow_version "15.15"
     completed true
     user
     project
     workflow
+    workflow_version { workflow ? "#{workflow.major_version}.#{workflow.minor_version}" : nil }
```

Comment:
Does this inherit from the workflow association on the line above? for some reason i assume this had to happen in the after build hooks, nice to know not.

Reply:
Yep, this works fine and even you pass in a specific workflow (as evidenced by the test failing on `create :classification, workflow: nil`)